content/network/drivers/bridge.md

```diff
@@ -133,6 +134,27 @@ your own `docker0` bridge. Use this option if you want to run multiple daemon
 instances on the same host. For details, see
 [Run multiple daemons](../../reference/cli/dockerd.md#run-multiple-daemons).
 
+### Default host binding address
+
+When no host address is given in port publishing options like `-p 80`
+or `-p 8080:80`, the default is to make the container's port 80 available on all
+host addresses, IPv4 and IPv6.
+
+Option `com.docker.network.bridge.host_binding_ipv4` can be used to modify the
+default address for published ports.
```

Comment:
suggestion: clarify (again) that this is a driver option, since we refer to
a CLI option in the previous paragraph.

```suggestion
The bridge network driver option `com.docker.network.bridge.host_binding_ipv4`
can be used to modify the default address for published ports.
```

Reply:
Done.